cmd/up.go

```diff
@@ -368,6 +352,60 @@ func newUpCmd() *cobra.Command {
 	return cmd
 }
 
+// handleConfig handles prompting for config values (as needed) and saving config.
+func handleConfig(
+	s backend.Stack,
+	templateNameOrURL string,
+	template workspace.Template,
+	configArray []string,
+	yes bool,
+	opts display.Options) error {
+
+	// Get the existing config. stackConfig will be nil if there wasn't a previous deployment.
+	stackConfig, err := backend.GetLatestConfiguration(commandContext(), s)
+	if err != nil && err != backend.ErrNoPreviousDeployment {
+		return err
+	}
+
+	// Get the existing snapshot.
+	snap, err := s.Snapshot(commandContext())
+	if err != nil {
+		return err
+	}
+
+	// Handle config.
+	// If this is an initial preconfigured empty stack (i.e. configured in the Pulumi Console),
+	// use its config without prompting.
+	// Otherwise, use the values specified on the command line and prompt for new values.
+	// If the stack already existed and had previous config, those values will be used as the defaults.
+	var c config.Map
+	if isPreconfiguredEmptyStack(templateNameOrURL, template.Config, stackConfig, snap) {
+		c = stackConfig
+		// TODO consider warning if the specific URL is different from templateURL.
```

Comment:
Can you file an issue to track this TODO and put in the number here?

Reply:
Done